### DISCUSSION TITLE
Cannot make app_internal_param_log

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#235] [Q&A] [answered]
Hello. I have successfully ran the app_hello_world example. I am now trying to run the app_internal_param_log example. But when making, I get this message:
= = = = = = = = = = = = = = 
make[2]: Leaving directory '/home/bitcraze/projects/crazyflie-firmware/examples/app_internal_param_log/build'
make[1]: *** [tools/kbuild/Makefile.kbuild:150: sub-make] Error 2
make[1]: Leaving directory '/home/bitcraze/projects/crazyflie-firmware'
make: *** [../../tools/make/oot.mk:23: all] Error 2
= = = = = = = = = = = = = = 
Please advise.

NOTE: the README file for app_internal_param_log says to make from the app_hello_world path. Is this a typo in the REAMDE?

**Comment by knmcguire** [comment, marked as answer]
Ah there is a mismatch with the kbuild file and the actual file name. You see that when you look more up in the build print outs:

```
make[4]: *** No rule to make target '/home/knmcguire/development/bitcraze/c/crazyflie-firmware/examples/app_internal_param_log/src/app_internal_param_log.o', needed by '/home/knmcguire/development/bitcraze/c/crazyflie-firmware/examples/app_internal_param_log/src/built-in.o'.  Stop.
make[3]: *** [/home/knmcguire/development/bitcraze/c/crazyflie-firmware/scripts/Makefile.build:311: /home/knmcguire/development/bitcraze/c/crazyflie-firmware/examples/app_internal_param_log/src] Error 2
make[2]: *** [tools/kbuild/Makefile.kbuild:589: /home/knmcguire/development/bitcraze/c/crazyflie-firmware/examples/app_internal_param_log] Error 2

```
I've created a PR for that here: https://github.com/bitcraze/crazyflie-firmware/pull/1102. You can see in there what I changed to fix that issue.

Thanks for reporting!

**Reply by Pandagineer** [reply]
This worked. Thanks! (I feel like such a hacker.) But note that the README.md file still says "Make sure that you are in the app_hello_world folder...", rather than "in the app_internal_param_log folder".

**Reply by knmcguire** [reply]
ah... we should actually really remove the readmes... they become lost and are not updated with the regular documentation. I've made a ticket so we are reminded of it: https://github.com/bitcraze/crazyflie-firmware/issues/1104

Thanks again ☺️
